.travis.yml

```diff
@@ -59,7 +59,7 @@ jobs:
         unzip -q master.zip;
         rm master.zip;
         cd NGSI-LD_TestSuite-master;  
-        export EXCLUDED_TESTS="001|003|042|080|081|126|129|130|149|";
+        export EXCLUDED_TESTS="001|003|042|080|081|126|129|149|123|";
```

Comment:
Has the test 123 been excluded on purpose?

Reply:
I have modified it